test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/AuthAndSessionDataBehaviour.scala

```diff
@@ -97,7 +98,7 @@ trait AuthAndSessionDataBehaviour { this: ControllerSpec with AuthSupport with J
 
     }
 
-  def authBehaviourWithPredicate(performAction: () => Future[Result]): Unit =
+  def authBehaviour(performAction: () => Future[Result]): Unit =
```

Comment:
Minor - I figure leaving this as `authBehaviour` is fine. `WithPredicate` suggests it takes a predicate as an argument.